docs/quickstart.md

```diff
@@ -1,73 +1,129 @@
 ## Prerequisites:
 
-In order to run the Postgres operator locally in minikube you need to install the following tools:
+In order to run the Postgres Operator locally in minikube you need to install
+the following tools:
 
 * [minikube](https://github.com/kubernetes/minikube/releases)
 * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl)
 
 Note that you can also use built-in Kubernetes support in the Docker Desktop
 for Mac to follow the steps of this tutorial. You would have to replace
-`minikube start` and `minikube delete` with your launch actionsfor the Docker
+`minikube start` and `minikube delete` with your launch actions for the Docker
 built-in Kubernetes support.
 
-## Local execution
+Clone the the repository and change to the directory. Then start minikube.
 
 ```bash
 git clone https://github.com/zalando/postgres-operator.git
 cd postgres-operator
 
 minikube start
+```
 
-# start the operator using one of helm chart or yaml manifests;
+## Manual deployment setup
 
-# - install postgres-operator with helm chart.
-# 1) initialize helm
-helm init
-# 2) install postgres-operator chart
-helm install --name postgres-operator ./charts/postgres-operator
+The Postgres Operator can be installed simply by applying yaml manifests.
 
-# - install postgres-operator with yaml manifests.
+```bash
 kubectl create -f manifests/configmap.yaml  # configuration
 kubectl create -f manifests/operator-service-account-rbac.yaml  # identity and permissions
 kubectl create -f manifests/postgres-operator.yaml  # deployment
+```
 
+## Helm chart
 
-# starting the operator may take a few seconds
-# check if operator pod is running
+Another possibility is using a provided [Helm](https://helm.sh/) chart which
+saves you these steps. Therefore, you would need to install the helm CLI on your
+machine. After initializing helm (and its server component Tiller) in your local
+cluster you can install the operator chart.
 
-# - if you've created the operator using helm chart
-kubectl get po -l app.kubernetes.io/name=postgres-operator
+```bash
+# 1) initialize helm
+helm init
+# 2) install postgres-operator chart
+helm install --name postgres-operator ./charts/postgres-operator
+```
+
+## Create a Postgres cluster
 
-# - if you've created the operator using yaml manifests
-kubectl get po -l name=postgres-operator
+Starting the operator may take a few seconds. Check if the operator pod is
+running before applying a Postgres cluster manifest.
+
+```bash
+# if you've created the operator using yaml manifests
+kubectl get pod -l name=postgres-operator
 
+# if you've created the operator using helm chart
+kubectl get pod -l app.kubernetes.io/name=postgres-operator
 
 # create a Postgres cluster
 kubectl create -f manifests/minimal-postgres-manifest.yaml
+```
+
+After the cluster manifest is submitted a StatefulSet will be created which
+rolls out the database instances (Pods and Services). They are named like the
```

Comment:
services and stateful set are separate k8s entities. Judging from our own diagram, there are also created separately

Reply:
true, description was slightly incorrect